video_xblock/static/js/videojs_event_plugin.js

```diff
@@ -0,0 +1,145 @@
+/**
+ * This part is responsible for tracking video player events.
+ * List of events:
+ * - onReady
+ * - onPlay
+ * - onPause
+ * - onEnded
+ * - onSeek
+ * - onSpeedChange
+ *
+ */
+(function () {
+
+    "use strict";
+    function XBlockEventPlugin(options) {
+
+        var previousTime = 0;
+        var currentTime = 0;
+
+        this.events = [
+            'onReady',
+            'onPlay',
+            'onPause',
+            'onEnded',
+            'onSpeedChange',
+             'onSeek',
+            /*TODO
+             'onShowLanguageMenu',
+             'onHideLanguageMenu',
+             'onShowTranscript',
+             'onHideTranscript',
+             'onShowCaptions',
+             'onHideCaptions',
+             */
+        ];
+
+        this.onReady = function () {
+            this.log('ready_video');
```

Comment:
This `_` and similar below concern me.
Are default player event titles use `_`, but not `-` or camelCase?